feat(content): adr for profile event keys (#804)
ockam-network/ockam: documentation/architecture/decisions/0002-profile-event-signing.md

# 2. Profile Event Signing

Date: 2021-02-02

## Status

Proposed

## Context
Profile events can include multiple keys so the profile can prove the keys are valid. The question centers on
when adding new keys, should a proof of possession be required? This involves the new key signing the current
event data, then the current profile root key signing the event data with the proof into the event.

## Decision

> It has been decided that a proof of possession will be required for now when adding a new key to the profile.

Reviewer: ```suggestion
Profile owners are required to provide proof of possession when adding a new key to the profile.
```
